### PR TITLE
Josh/adding env to my file

### PR DESCRIPTION
**Pull request by JoshuaCheney**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+NODE_ENV="development"
+REACT_APP_PORT=3000
+
+REACT_APP_FETCH_URL="http://localhost:3030"
```
Comment (Auenc) [inline .env:4]: You don't want to push your `.env` file to your git repo! Add a rule in your `.gitignore` so git will ignore it :)

Comment (JoshuaCheney) [reply .env:4]: its in there

**Review by Auenc — CHANGES_REQUESTED** [review]
Looks great, but you want to make sure you're not pushing your `.env` file

File: .env (hunk 1 above)
Comment (Auenc) [reply .env:4]: Did you add your `.gitignore` to the commit? There doesn't seem to be able changes in your `.gitignore` in this pull request

**Pull request merged**